status.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import json
+import requests
+
+from os import path
+from typing import Any, Dict
+
+from constants import base_payload, released_premined_tokens, server, total_premined
+
+
+def run(event: Dict[str, Any], context: 'bootstrap.LambdaContext') -> Dict[str, Any]:
+    """ Get mined tokens from the full node and return the economics status of the network
+    """
+    # Get mined tokens from full node
+    response = requests.get(path.join(server, 'mined_tokens'))
+    if response and response.status_code == 200 and response.json().get('mined_tokens'):
```

Comment:
I think this endpoint has changed.

Reply:
Why do we check `response.json().get('mined_tokens')`? Shouldn't we check `response.json().get('success')`?

Reply:
Yes, it has changed. Thank

About the `response.json().get('mined_tokens')` is just for the test right now before deploying the full node.

If I check only `response.json().get('success')` it would enter the `if` statement but wouldn't have the mined_tokens data.

I will write a # FIXME to remember to remove it when it's done.

Reply:
ok